src/platform/plugins/shared/dashboard/public/plugin.tsx

```diff
@@ -340,13 +312,12 @@ export class DashboardPlugin
     untilPluginStartServicesReady().then(() => {
       registerActions({
         plugins,
-        allowByValueEmbeddables: this.dashboardFeatureFlagConfig?.allowByValueEmbeddables,
+        allowByValueEmbeddables:
```

Comment:
Linking [this](https://github.com/elastic/kibana/issues/137197) here as we should't forget to remove this setting.